.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+Thanks for opening an issue for us!  Before you open an issue, please check if a similar issue exists or has been closed before.
+
+If you're asking a question about how to use Blockly in your application, please ask questions on the [mailing list](https://groups.google.com/forum/#!forum/blockly) instead of filing issues.
+
+_Delete the above section and the instructions in the sections below before submitting.  If you don't do this, we'll assume you didn't bother to read the instructions or try to solve the problem before filing an issue.  If a section is not applicable, type "N/A" in that section._
```

Comment:
Add "Please" -> "Please delete the above..."

Remove the "If you don't do this" sentence. It will only annoy the people who are reading the instructions.

Reply:
Done.